### PR TITLE
Navigate to in app location for query results

### PR DESCRIPTION
**Pull request by dhaneshnm**
Parse urls in query results and point them to an in app location, if
available.

Fixes #BlueBrain/nexus/issues/838

### CODE REVIEW

**Comment by dhaneshnm** [comment]
To test, click on the resources or project urls in query result table. They should navigate to resources and projects paths respectively.

File: src/shared/components/ViewForm/SparqlQueryResults.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
 } from '@bbp/nexus-sdk';
 
 import './view-form.less';
+import { parseProjectUrl } from '../../utils';
```
Comment (Stafeeva) [inline src/shared/components/ViewForm/SparqlQueryResults.tsx]: is it used anywhere? :)

File: src/shared/components/ViewForm/SparqlQueryResults.tsx [hunk 2]
```diff
@@ -23,6 +24,34 @@ export type Entry = {
...
+      const [, org, proj] = entry.match(projectUrlPattern) as string[];
+      return `${org}/${proj}`;
+    }
+    if (resourceUrlPattern.test(entry)) {
+      const resourceIdPattern = /_\/([\w-|\W-]+)/;
+      const labels = entry.match(resourceUrlPattern) as string[];
+      if (resourceIdPattern.test(entry)) {
+        const resultArray = entry.match(resourceIdPattern) as string[];
+        if (resultArray !== null && resultArray.length > 1) {
+          console.log(`${labels[1]}/resources/${resultArray[1]}`);
```
Comment (Stafeeva) [inline src/shared/components/ViewForm/SparqlQueryResults.tsx]: I think you forgot to remove logging

File: src/shared/components/ViewForm/SparqlQueryResults.tsx [hunk 3]
```diff
@@ -23,6 +24,34 @@ export type Entry = {
   type: string;
 };
 
+const getUrl = (entry: string) => {
```
Comment (Stafeeva) [inline src/shared/components/ViewForm/SparqlQueryResults.tsx]: we could probably move the function to utils file and test it?

Comment (dhaneshnm) [reply src/shared/components/ViewForm/SparqlQueryResults.tsx]: I have move the bulk of logic to utils/added tests for it.

**Pull request merged**